src/foam/java/Method.js

```diff
@@ -73,6 +83,25 @@ foam.CLASS({
         o.indent();
         o.out('}');
       }
+
+      if( this.documentation ){
+        o.decreaseIndent();
+      }
+
     }
   ]
 });
+
+
+function limitSplit(o, str, maxWords){
```

Comment:
third time we have this method defined

Reply:
yes, just for testing, will be unified in a util file